cmd/console-proxy/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "humility-cmd-console-proxy"
+version = "0.1.0"
+edition = "2021"
+description = "SP/host console uart proxy"
+
+[dependencies]
+anyhow = { workspace = true }
+clap = { workspace = true }
+crossbeam-channel = { workspace = true }
+log = { workspace = true }
+parse_int = { workspace = true }
+
+humility = { workspace = true }
+humility-cmd = { workspace = true }
+cmd-hiffy = { workspace = true }
+
+hif = { workspace = true }
+idol = { workspace = true }
+
+[target.'cfg(not(windows))'.dependencies]
+termios = "0.3"
```

Comment:
Is there a way to make this a conditional workspace dependency?

Reply:
Maaaaybe? I don't think we can have target-specific workspace dependencies (e.g., https://github.com/rust-lang/cargo/issues/5220), but maybe we can specify `termios = "0.3"` as a workspace dep but conditionally depend on it in Cargo.toml. Trying that in 5db1630 to see if it passes Windows CI.

Reply:
I'd lean toward making them all workspace deps, because otherwise if I want to add a new dep, I first ought to grep and see if it's used by something else, and if so, move it to be a workspace dep now that it's becoming shared.

That's slightly annoying but not the end of the world; are there similar annoyances with "everything at the workspace level" I'm not thinking of?